packages/create-redwood-app/README.md

```diff
@@ -85,19 +85,25 @@ The installation script is built with [Yargs](https://github.com/yargs/yargs)
 The project codebase in `template/` uses [Yarn Workspace v1](https://classic.yarnpkg.com/en/docs/workspaces/) for a monorepo project containing the API and Web Sides. Redwood packages are included in `template/package.json`, `template/web/package.json`, and `template/api/package.json`, respectively.
 
 ### How to run create-redwood-app from your local repo and create a project
-Make sure you to first run `yarn install` in your project root.
+First, run `yarn install` in the root of the mono repo. Then, run `yarn install` again, inside the `packages/create-redwood-app` directory.
```

Comment:
A bit of a nit: I've always seen it as one word but don't have a source as to which is right

```suggestion
First, run `yarn install` in the root of the monorepo. Then, run `yarn install` again, inside the `packages/create-redwood-app` directory.
```